docs/security/principles/01-locked-vault-is-secure.mdx

```diff
@@ -0,0 +1,13 @@
+# P01 - A locked vault is secure
+
+Clients must ensure that once the vault has been locked, no vault data can be accessed in plain
```

Comment:
How do we reconcile use of obligatory language in this section ("must") with the fact that it's a non-actionable set of guidelines? Is this what the RFC means when it says:

>    Note that the force of these words is modified by the requirement
   level of the document in which they are used.

Reply:
Yeah exactly, principles are guiding stars, they describe general directions and ideals, but in the end it's the requirement that you test against and so that's the language you go after.

When I wrote these I meant to set high standards to encourage us to find new ways of protecting the user. As our technology improves we should be able to use these principles as justification for new and stricter requirements. Because of this I'm hesitant to add loopholes here because some platforms might actually give us the tools necessary to achieve these principles